tests/mk_test.sh

```diff
@@ -155,6 +155,22 @@ function test_mk_build()
   expected_result="make -j$PARALLEL_CORES ARCH=x86_64 "
   assertEquals "($ID)" "$expected_result" "${output}"
 
+  ID=7
```

Comment:
We replaced `ID` with `LINENO`.

Reply:
I left the `ID` as a comment, a lot of the tests have not been updated to use `LINENO` and I think a bigger refactor should be done to update these tests.